ml_pipeline_gen/pipelines.py

```diff
@@ -226,6 +230,43 @@ def _write_template(self, env, template_path, args, dest):
         with open(dest, "w+") as f:
             f.write(body)
 
+    def configure_kfp_cluster(self):
+        """Calls shell script to create a KFP cluster."""
+        model = self.model
+        subprocess.call([
+            "bin/create_kfp_cluster.sh",
+            model.project_id,
+            model.cluster_name,
+            model.zone
+        ])
+
+    def get_kfp_hostname(self):
+        """Returns Hostname (URL) of KFP cluster in current kube context."""
+        # Checks default kubectl context from ~/.kube/config
+        config.load_kube_config()
+
+        name = "inverse-proxy-config"
+        namespace = "kubeflow"
+
+        instance = client.CoreV1Api()
+
+        response = instance.read_namespaced_config_map(name, namespace)
+        while response.data is None:
+            print("Waiting for KFP Dashboard to be created...")
+            time.sleep(60)
+            try:
+                response = instance.read_namespaced_config_map(name, namespace)
+            except ApiException as e:
+                print("Exception -> CoreV1Api: %s\n" % e)
+
+        # Although hostname is available, accessing it can throw 502/504s if
+        # the dashboard isn't fully configured. Hence, wait for 180 sec.
+        time.sleep(180)
```

Comment:
Could we just warn the user the dashboard might not be available yet instead of sleeping for an arbitrary amount of time?